src/main/java/com/jalasoft/sdfc/pages/AppLaunchPage.java

```diff
@@ -0,0 +1,23 @@
+package com.jalasoft.sdfc.pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public class AppLaunchPage extends AbstractPage {
+    private static final String PRIVACY_CSS = "//a[@title='%s']";
+    @FindBy(css = ".slds-icon-waffle_container")
+    private WebElement AppLauncherButton;
+
+    @FindBy(css = "#password")
+    private WebElement passwordTextField;
+
+    @FindBy(css = "#Login")
+    private WebElement loginButton;
+
+    public Header LaunchApp(String feature) {
```

Comment:
As far as I can understand, this method click items and not apps, so I think this should be called `clickOnItem(String item)` or `launchItem(String item)`

Why this method returns a Header?
it is expected to use the Header page object all the time once an item is clicked?
I think this should be a `void` method instead

Reply:
please review the last commit